python/modules/xgbHelper.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python 
+import os, sys 
+import ROOT 
+ROOT.PyConfig.IgnoreCommandLineOptions = True 
+from importlib import import_module 
+import numpy as np 
+import xgboost as xgb 
+import logging 
+
+class XGBHelper:     
```

Comment:
This is a small class.  Can you move it to the tauMVA since it is only used there?

Reply:
Done